cmd/ddev/cmd/describe_test.go

```diff
@@ -87,6 +88,16 @@ func TestDescribeAppFunction(t *testing.T) {
 		assert.Contains(string(out), app.GetName())
 		assert.Contains(string(out), "running")
 		assert.Contains(string(out), v.Dir)
+		assert.Contains(string(out), "DDEV ROUTER STATUS: "+color.CyanString("running"))
```

Comment:
This seems a little fragile; how about a regex match that doesn't look for the color? (I won't insist)